web/cypress/support/page_objects/landingPage.ts

```diff
@@ -1,14 +1,19 @@
+import { useConfig } from "@/lib/data-hooks/useConfig";
+
+const { Config } = useConfig();
+let landingPageConfig = Config.landingPage;
+
 class LandingPage {
   getHowItWorksSection = () => {
-    return cy.get('[aria-label="How it works"]');
+    return cy.get('[aria-label="' + `${landingPageConfig.section4HeaderText}` + '"]');
```

Comment:
@cbrunefearless why not use a backtick instead of using single quotes and adding them together, I. think it would be more readable